cmd/flags.go

```diff
@@ -372,3 +373,40 @@ func duration(s string) time.Duration {
 	}
 	return d + time.Hour*time.Duration(v*24)
 }
+
+func parseBytes(ctx *cli.Context, key string, unit byte) uint64 {
+	str := ctx.String(key)
+	if len(str) == 0 {
+		return 0
+	}
+
+	s := str
```

Comment:
use humanize.ParseBytes(ctc.String(key)) ?

Reply:
For simplicity, I want to use a single char as the unit, like 'M' or 'G'. While in the humanize package, you have to write 'mi' or 'gi'.

Reply:
docker/go-units RAMInBytes support that
https://github.com/docker/go-units/blob/16e18b2861ca6fd622e7042ffeb9a3ebe8a9dff9/size.go#L86